src/windows/exportClips.py

```diff
@@ -0,0 +1,236 @@
+"""
```

Comment:
This file should be all lower-case, with _ between words (like the other window classes)